src/app/features/create-event/create-event.service.ts

```diff
@@ -1,69 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable, inject } from '@angular/core';
+import { API_URL } from '@core/env.token';
+import { FetchedGroup } from './create-event.interface';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root',
+})
 export class CreateEventService {
-  private builder = inject(NonNullableFormBuilder);
-  private eventForm = this.createForm();
+  private API_URL = inject(API_URL);
+  private http = inject(HttpClient);
 
-  getForm(): FormGroup<EventForm> {
-    return this.eventForm;
-  }
-
-  private createForm() {
-    return this.builder.group<EventForm>({
-      eventTypeForm: this.builder.group({
-        isInternal: this.builder.control(false),
-        isPrivate: this.builder.control(false),
-        isGroup: this.builder.control(false),
-        isExternal: this.builder.control(false),
-      }),
-      eventDetailsForm: this.builder.group(
-        {
-          group: this.builder.control<number | null>(null),
-          limitedPlaces: this.builder.control(
-            { value: 1, disabled: true },
-            {
-              validators: [Validators.required, Validators.min(1), Validators.max(10000)],
-            }
-          ),
-          isLimitedPlaces: this.builder.control(false),
-          isConfirmationRequired: this.builder.control(false),
-          name: this.builder.control('', {
-            validators: [Validators.required, Validators.minLength(4), Validators.maxLength(100)],
-          }),
-          category: this.builder.control<string[]>([]),
-          startDate: this.builder.control('', {
-            validators: [Validators.required],
-          }),
-          hour: this.builder.control('', {
-            validators: [Validators.required],
-          }),
-          postCode: this.builder.control('', {
-            validators: [Validators.required, Validators.pattern(patterns.postCode)],
-          }),
-          city: this.builder.control('', {
-            validators: [Validators.required, Validators.minLength(2), Validators.maxLength(30)],
-          }),
-          street: this.builder.control('', {
-            validators: [Validators.required, Validators.minLength(3), Validators.maxLength(60)],
-          }),
-          streetNumber: this.builder.control('', {
-            validators: [Validators.required, Validators.maxLength(10)],
-          }),
-          apartmentNumber: this.builder.control('', {
-            validators: [Validators.maxLength(5)],
-          }),
-          description: this.builder.control('', {
-            validators: [Validators.required, Validators.minLength(4), Validators.maxLength(4000)],
-          }),
-          hashtags: this.builder.control<string[]>([]),
-        },
-        { validators: isHourInThePastValidator }
-      ),
-    });
+  fetchUserGroups() {
+    // this endpoint will be replaced with endpoint that contains user id
+    return this.http.get<FetchedGroup[]>(`${this.API_URL}groups`);
```

Comment:
nie zjadłeś `/` przed groups? url api to przykładowo `http://localhost:3000` bez `/` na końcu